js/order.js

```diff
@@ -1,8 +1,13 @@
 export const OrderAPI = superclass =>
     class extends superclass {
-        async getOrder(number) {
+        async getOrder(number, options = {}) {

```

Comment:
This should be a dumb API client, meaning that you just `this.get(url, options)` without changing them. Clients of the API will have to build the options to contain a `X-Secret-Key` header by themselves.